src/Game/DLC/aocChampionBalladManager.cpp

```diff
@@ -0,0 +1,34 @@
+#include "Game/DLC/aocChampionBalladManager.h"
+
+#include <math/seadMathCalcCommon.h>
+
+namespace uking {
+
+SEAD_SINGLETON_DISPOSER_IMPL(ChampionBalladManager);
```

Comment:
```suggestion
SEAD_SINGLETON_DISPOSER_IMPL(ChampionBalladManager)
```

SEAD_SINGLETON_DISPOSER_IMPL expands to a bunch of function definitions and typically people don't put semicolons after the closing brace of a function body.